qml/components-v-play/Image.qml

```diff
@@ -0,0 +1,15 @@
+import QtQuick 2.7
+import VPlayApps 1.0
+
+Image {
+
+    MouseArea {
+        anchors.fill: parent
+        onClicked: {
+            // do not make the place holder image fullscreen
+            if (!source.toString().includes("Platzhalter_v2_mitSchriftzug")) {
+                 PictureViewer.show(app, source)
```

Comment:
Damn that they don't offer a way to load all the pictures and swipe through the list...